odrive-motor/src/odriveCAN.py

```diff
@@ -197,9 +201,9 @@ async def is_powered(self, extra: Optional[Dict[str, Any]] = None, **kwargs) ->
 
     async def is_moving(self) -> bool:
         for msg in bus:
-            if msg.arbitration_id == ((self.nodeID << 5) | db.get_message_by_name('Get_Encoder_Estimates').frame_id):
-                current_vel = db.decode_message('Get_Encoder_Estimates', msg.data)['Vel_Estimate']
-                if current_vel > 0:
+            if msg.arbitration_id == ((self.nodeID << 5) | 0x009):
```

Comment:
[q] Why did this need to be passed in the raw value rather than go through `get_message_by_name`?

Reply:
oops it doesn't. think I copied from somewhere in my tests